Signal/src/Calls/UserInterface/CallControls.swift

```diff
@@ -192,8 +201,8 @@ class CallControls: UIView {
         hangUpButton.isHidden = call.groupCall.localDeviceState.joinState != .joined
 
         // Use small controls if video is enabled and we have external
-        // audio inputs, because we have five buttons now.
-        [audioSourceButton, flipCameraButton, videoButton, muteButton, hangUpButton].forEach {
+        // audio inputs, because we have six buttons now (generally only on iPad, and 5 buttons for iPhone, due to center stage).
+        [audioSourceButton, flipCameraButton, videoButton, centerStageButton, muteButton, hangUpButton].forEach {
```

Comment:
I think you'll want to check if center stage is supported to see how many buttons are actually being rendered (you can check the `isHidden` property on the button). The buttons probably don't need to be small to fit if the `centerStageButton` is hidden.

Reply:
I think I checked the hidden property in another place in this PR?